compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.h

```diff
@@ -20,6 +20,28 @@
 #include "mlir/IR/BuiltinAttributes.h"
 #include "mlir/IR/BuiltinTypes.h"
 
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace GPU {
```

Comment:
nit: Convention is single line namespacing
```suggestion
namespace mlir::iree_compiler::IREE::GPU {
```